config/auth_default.go

```diff
@@ -15,11 +15,15 @@ var (
 		BasicCredentials{},
 		M2mCredentials{},
 		BricksCliCredentials{},
+
+		// Attempt to configure the more specific auth methods before calling into the CLI.
 		AzureMsiCredentials{},
 		AzureClientSecretCredentials{},
 		AzureCliCredentials{},
-		GoogleDefaultCredentials{},
+
+		// Attempt to configure the more specific auth methods before calling into the CLI.
```

Comment:
`GoogleDefaultCredentials` does not call into the `gcloud` CLI. it just reads directly from `~/.config/gcloud/application_default_credentials.json` if it's present on the system, which makes this comment incorrect.

Reply:
Thanks, fixed.